### PR TITLE
add maximum-paths to bgp_af

### PR DESCRIPTION
**Pull request by andy91315**
1. pass beaker test
Please help me check the code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -369,6 +369,10 @@ Sub Address Family Identifier (SAFI). Required. Valid values are `unicast` and `
 ##### `default_information_originate`
 `default-information originate`. Valid values are true and false.
 
+##### `maximum_paths [ibgp]`
```
Comment (chrisvanheuveln) [inline README.md]: Don't combine these. This should have an entry for each property, ie>

```
##### `maximum_paths`
  blah

##### `maximum_paths_ibgp`
  blah
```

File: README.md [hunk 2]
```diff
@@ -369,6 +369,10 @@ Sub Address Family Identifier (SAFI). Required. Valid values are `unicast` and `
...
+Configures the maximum number of equal-cost paths for load sharing. Valid value are intergers between 1 and 64.
```
Comment (chrisvanheuveln) [inline README.md]: /are intergers/is an integer/

File: lib/puppet/type/cisco_bgp_af.rb [hunk 3]
```diff
@@ -196,6 +198,34 @@ def self.title_patterns
     newvalues(:true, :false, :default)
   end # property :default_information_originate
 
+  newproperty(:maximum_paths) do
+    desc "Configures the maximum number of equal-cost paths for load sharing.
+          Valid values are integers between 1 and 64, default value is 1."
```
Comment (chrisvanheuveln) [inline lib/puppet/type/cisco_bgp_af.rb]: /Valid values are integers between 1 and 64/Valid value is an integer in the range 1 - 64/

File: lib/puppet/type/cisco_bgp_af.rb [hunk 4]
```diff
@@ -196,6 +198,34 @@ def self.title_patterns
...
+    munge do |value|
+      value = :default if value == 'default'
+      unless value == :default
+        value = value.to_i
+        fail 'maximum_paths value should be between 1 and 64' unless
```
Comment (chrisvanheuveln) [inline lib/puppet/type/cisco_bgp_af.rb]: /between 1 and 64/in the range 1 - 64/

File: lib/puppet/type/cisco_bgp_af.rb [hunk 5]
```diff
@@ -196,6 +198,34 @@ def self.title_patterns
...
+  end # property :maximum_paths
+
+  newproperty(:maximum_paths_ibgp) do
+    desc "Configures the maximum number of equal-cost paths for load sharing.
+          Valid values are integers between 1 and 64, default value is 1."
+    munge do |value|
+      value = :default if value == 'default'
+      unless value == :default
+        value = value.to_i
+        fail 'maximum_paths_ibgp value should be between 1 and 64' unless
```
Comment (chrisvanheuveln) [inline lib/puppet/type/cisco_bgp_af.rb]: /between 1 and 64/in the range 1 - 64/

File: lib/puppet/type/cisco_bgp_af.rb [hunk 6]
```diff
@@ -196,6 +198,34 @@ def self.title_patterns
...
+        fail 'maximum_paths value should be between 1 and 64' unless
+          value.between?(1, 64)
+      end
+      value
+    end
+  end # property :maximum_paths
+
+  newproperty(:maximum_paths_ibgp) do
+    desc "Configures the maximum number of equal-cost paths for load sharing.
+          Valid values are integers between 1 and 64, default value is 1."
```
Comment (chrisvanheuveln) [inline lib/puppet/type/cisco_bgp_af.rb]: /Valid values are integers between 1 and 64/Valid value is an integer in the range 1 - 64/

File: lib/puppet/type/cisco_bgp_af.rb [hunk 7]
```diff
@@ -196,6 +198,34 @@ def self.title_patterns
...
+        value = value.to_i
+        fail 'maximum_paths value should be between 1 and 64' unless
+          value.between?(1, 64)
+      end
+      value
+    end
+  end # property :maximum_paths
+
+  newproperty(:maximum_paths_ibgp) do
+    desc "Configures the maximum number of equal-cost paths for load sharing.
```
Comment (chrisvanheuveln) [inline lib/puppet/type/cisco_bgp_af.rb]: /number of equal-cost/number of ibgp equal-cost/

**Comment by andy91315** [comment]
change the message in comment

File: README.md [hunk 8]
```diff
@@ -369,6 +369,14 @@ Sub Address Family Identifier (SAFI). Required. Valid values are `unicast` and `
 ##### `default_information_originate`
 `default-information originate`. Valid values are true and false.
 
+##### `maximum_path`
```
Comment (chrisvanheuveln) [inline README.md]: You dropped the 's'. The rest is fine though so I'll fix it post-merge.

**Pull request merged**